paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -123,9 +123,29 @@ internal class CustomerSheetViewModel @Inject constructor(
         TODO()
     }
 
-    @Suppress("UNUSED_PARAMETER")
     private fun onItemSelected(paymentSelection: PaymentSelection?) {
-        TODO()
+        when (paymentSelection) {
+            is PaymentSelection.GooglePay, is PaymentSelection.Saved -> {
+                updateViewState<CustomerSheetViewState.SelectPaymentMethod> {
+                    it.copy(
```

Comment:
Do we want to clear errors or anything when they select a new option?

Reply:
I thought about this, and given that the only error source right now is when the payment methods cannot be loaded, there is no way for this to be called. Added a comment though to remember to clear the error message if we do end up emitting other error messages.